x-pack/plugins/ml/public/data_frame/components/source_index_preview/use_source_index_data.ts

```diff
@@ -39,40 +40,38 @@ export const useSourceIndexData = (
   const [status, setStatus] = useState(SOURCE_INDEX_STATUS.UNUSED);
   const [tableItems, setTableItems] = useState([] as EsDoc[]);
 
-  if (indexPattern !== null) {
-    const getSourceIndexData = async function() {
-      setErrorMessage('');
-      setStatus(SOURCE_INDEX_STATUS.LOADING);
-
-      try {
-        const resp: SearchResponse<any> = await ml.esSearch({
-          index: indexPattern.title,
-          size: SEARCH_SIZE,
-          body: { query },
-        });
+  const getSourceIndexData = async function() {
+    setErrorMessage('');
+    setStatus(SOURCE_INDEX_STATUS.LOADING);
 
-        const docs = resp.hits.hits;
+    try {
+      const resp: SearchResponse<any> = await ml.esSearch({
```

Comment:
not that it really makes a difference, but there is a type for this `SearchResponse` from `elasticsearch`